jailer.go

```diff
@@ -358,6 +358,7 @@ func jail(ctx context.Context, m *Machine, cfg *Config) error {
 
 	m.cmd = builder.Build(ctx)
 
+	m.Handlers.FcInit = m.Handlers.FcInit.Swap(createBootstrapLoggingHandler(m.Cfg.LogFifo))
```

Comment:
I'm not super familiar with this aspect of the gosdk, but would it make more sense to put this swap in the call to `ChrootStrategy.AdaptHandlers` call below? On the surface `AdaptHandlers` seems like the right place to update the bootstrapLoggingHandler's log fifo path.

Reply:
Changed to just how default handlers are constructed